src/core/transaction.js

```diff
@@ -1112,6 +1112,77 @@ Transaction.prototype.setDeliverMin = function(deliver_min) {
   return this._setAmount('DeliverMin', deliver_min);
 };
 
+Transaction.prototype.suspendedPayment = function(options_) {
+  const options = _.merge({}, options_);
```

Comment:
Oh, I see you're allowing `cancelAfter` and `cancel_after`, and don't want to mutate the passed in options_.